dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/FactSet.Protobuf.Stach.Extensions.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you please add the metadata similar to this [SDK](https://github.com/factset/stachschema-sdks/blob/master/dotnet/Schema/Schema.csproj#L1)?